lib/app/features/competiton/searched_results/result_details/result_details_view.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_modular/flutter_modular.dart';
+import 'package:score_live/app/common/extensions/context/applocalization_context.dart';
+import 'package:score_live/app/features/competiton/searched_results/result_details/cubit/result_details_cubit.dart';
+import 'package:score_live/app/features/competiton/searched_results/result_details/widgets/result_details_header.dart';
+import 'package:score_live/app/features/competiton/searched_results/result_details/widgets/result_details_tab_bar.dart';
+import 'package:score_live/app/features/competiton/searched_results/result_details/widgets/tabs/fixtures_tab.dart';
+import 'package:score_live/app/features/competiton/searched_results/result_details/widgets/tabs/results_tab.dart';
+import 'package:score_live/app/features/competiton/searched_results/result_details/widgets/tabs/standings_tab.dart';
+import 'package:score_live/app/features/competiton/searched_results/result_details/widgets/tabs/stats_tab.dart';
+import 'package:score_live/presentation/constants/app_colors.dart';
+import 'package:score_live/presentation/constants/text_styles.dart';
+
+import '../../../../common/extensions/enums.dart';
+import '../../../../common/widgets/custom_app_bar.dart';
+import '../../../../common/widgets/search_list_tile.dart';
+
+class ResultDetailsView extends StatelessWidget {
+  const ResultDetailsView(
+    this.params, {
+    super.key,
+  });
+
+  final ResultParams params;
+
+  @override
+  Widget build(BuildContext context) {
+    const space = 20.0;
+    return Scaffold(
+      backgroundColor: AppColors.backgroundBlack,
+      appBar: CustomAppBar(
+        title: Center(
+          child: Text(
+            context.localizations.navCompetition,
+            style: const CustomTextStyle(
+              fontSize: 16,
+              color: Colors.white,
+              fontWeight: FontWeight.w700,
+            ),
+          ),
+        ),
+        leading: IconButton(
+          onPressed: () => Modular.to.pop(),
+          icon: const Icon(Icons.arrow_back_ios, color: Colors.white, size: 24),
+        ),
+        actions: [
+          IconButton(onPressed: () {}, icon: const Icon(Icons.share, color: Colors.white, size: 20)),
```

Comment:
Why do these two icons have their `onPressed` methods empty? If you plan on using them in the future, please add a `TODO`, otherwise show only the Icon